### PR TITLE
testing env for window3

### PR DESCRIPTION
**Pull request by TotoFullStackDev**
##### Description
As usual, you shall setup custom RPC on metamask, using this fork
https://rpc.tenderly.co/fork/46955249-a8a9-4274-83eb-6c98fe090a61

##### Checklist

- [ ] Linter status: 100% pass
- [ ] Changes don't break existing behavior
- [ ] Test coverage hasn't decreased

### CODE REVIEW

File: src/config/env.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "networkId": 1
   },
   "customRPC": {
-    "url": "https://rpc.tenderly.co/fork/112e7699-fa6c-4f5d-b184-10627f12235c",
-    "enabled": false
+    "url": "https://rpc.tenderly.co/fork/92a699c6-2af6-433e-b7c3-108241c427cb",
+    "enabled": true
```
Comment (TotoFullStackDev) [inline src/config/env.json]: REMINDER: set this to false before merge

File: src/stores/eth/connection.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ import env from '../../config/env.json';
 export const defaultProvider = env.blocknative.networkId === 1
   ? new ethers.providers.InfuraProvider(
     'homestead', 
-    'e106b2b27c0f4941be1f2c183a20b3ea', // production key
-    // '1ec103a49691457aa6dff30aa8ab73d0' // testing key
+    // 'e106b2b27c0f4941be1f2c183a20b3ea', // production key
+    '1ec103a49691457aa6dff30aa8ab73d0' // testing key
```
Comment (TotoFullStackDev) [inline src/stores/eth/connection.js]: REMINDER: change infura key before merge

**Review by AdrianAddesso — CHANGES_REQUESTED** [review]
Can't claim, console issue below:

image

**Review by AdrianAddesso — APPROVED** [review]
Claim and redemption worked properly

**Review by dantop114 — APPROVED** [review]

**Comment by Alexintosh** [comment]
lgtm

**Review by Alexintosh — APPROVED** [review]

**Pull request merged**